packages/amplify-provider-awscloudformation/src/configuration-manager.js

```diff
@@ -25,7 +24,12 @@ async function init(context) {
   }
   normalizeInputParams(context);
 
-  const appId = resolveAppId(context);
+  let appId;
+  try {
```

Comment:
is this try necessary even with the changes in resolveAppId?

Reply:
Yes. It will still throw if an appId isn't found. This fixes `amplify pull` in an empty dir.